static/header.less

```diff
@@ -59,7 +59,7 @@
         .dropdown {
             position: absolute;
             top: 52px;
-            z-index: 91;
+            z-index: @z-index-mid;
```

Comment:
This one doesn’t seem necessary either because the header already has a z-index.